src/gym_electric_motor/utils.py

```diff
@@ -2,18 +2,17 @@
 import numpy as np
 
 
-# This hacky function stays for now because, all envs use it and refectoring would be a pain TODO: fix this
+# This hacky function stays for now because all envs uses it and refectoring would be a pain
```

Comment:
Can we elaborate on this more? Either remove this comment and raise an issue explaining why this function should be removed and how it is difficult to do so. Otherwise, please make this comment more elaborate (also make it sound more neutral). Thank you